oshdb-util/src/test/java/org/heigit/bigspatialdata/oshdb/util/time/OSHDBTimestampsTest.java

```diff
@@ -0,0 +1,61 @@
+package org.heigit.bigspatialdata.oshdb.util.time;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.SortedSet;
+import org.heigit.bigspatialdata.oshdb.util.OSHDBTimestamp;
+import org.heigit.bigspatialdata.oshdb.util.time.OSHDBTimestamps.Interval;
+import org.junit.Test;
+
+public class OSHDBTimestampsTest {
+
+  @Test
+  public void testTimeIntervals() {
+    List<List<Object>> inputs = new ArrayList<>();
```

Comment:
Maybe it would be nicer to change this into separate lists for for from/to/expected timestamps and intervals. Or to use fewer variables, use one `Map<Interval, List<String>>` for from/to timestamps and one `Map<Interval, List<String>>` for expected results?

Reply:
Chose the variant with multiple lists because otherwise it wouldn't be possible to add multiple test cases for the same interval.